modules/ROOT/pages/gatekeeper.adoc

```diff
@@ -6,6 +6,21 @@ endif::[]
 
 When you deploy an application, the tracked resources exposed by the application might become accessible for a short amount of time due to a lapse in security. Because the policies for those resources are not yet applied, the API might receive unwanted traffic. GateKeeper prevents this situation by blocking the tracked resource until all policies are retrieved and applied without errors. 
 
+Starting with CloudHub 4.2.2, the Mule runtime engine (Mule) keep APIs blocked until the contracts of the API are successfully downloaded from the Management plane in case the policy requires contracts and if Gatekeeper is enabled. Gatekeeper waits until contracts are available to unblock the API.
```

Comment:
Suggestion: The sentence is long. Although I usually don't like parentheses, I think putting "in case the policy requires contracts" in parentheses makes the important parts easier to read.